http.go

```diff
@@ -270,6 +272,22 @@ func (hc *HttpCall) ResponseBodyContains(value string) Step {
 	})
 }
 
+// ResponseBodyMatches is a Step that when executed ensures there is
+// a non-nil hc.ResponseBody and errors unless the hc.ResponseBody
+// matches the regular expression parameter.
+func (hc *HttpCall) ResponseBodyMatches(pattern string) Step {
```

Comment:
Thanks for the PR. In the interests of flexibility, I would prefer this to take a *Regexp rather than a string.